src/components/organisms/Home.tsx

```diff
@@ -1,43 +1,47 @@
+import { useState } from "react";
+import useLottery from "../../hooks/useLottery";
 import Dropdown from "../atoms/Dropdown/Dropdown";
 import Footer from "../atoms/Footer/Footer";
+import { Icon } from "../atoms/Icons/Icons";
 import Results from "../molecules/Results/Results";
-
+import { useThemeContext } from "../../providers/theme/Context";
 import "./style.css";
 
 const Home = (): JSX.Element => {
+  const [results, setResults] = useState<string[]>([]);
+  const { getResult, options } = useLottery();
+  const { lottery, setLottery } = useThemeContext();
+
+  const choosedLottery: Record<string, string> = {
```

Comment:
Instead of creating this `choosedLottery` here you can use `lotteries` from `useLottery`